sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluate/_utils.py

```diff
@@ -191,6 +192,7 @@ def _log_metrics_and_instance_results_onedp(
             evaluation=EvaluationUpload(
                 display_name=evaluation_name,
                 properties=properties,
+                tags=tags,
```

Comment:
This will require service side support. I will tag few folks from service to check if it is supported

Reply:
i did run the sample `aoai_score_model_grader_sample.py` with the `AZURE_AI_PROJECT_ENDPOINT`. And that run did registry the tags

<img width="914" height="368" alt="image" src="https://github.com/user-attachments/assets/2e8991a5-4e2e-48c0-a7a6-2479684b95d8" />